docs/ops/filesystems/index.md

```diff
@@ -35,32 +35,50 @@ File system instances are instantiated once per process and then cached/pooled,
 * This will be replaced by the TOC
 {:toc}
 
-## Built-in File Systems
+## Local File System
 
-Flink ships with implementations for the following file systems:
+Flink has built-in support for the file system of the local machine, including any NFS or SAN drives mounted into that local file system.
+It can be used by default without additional configuration. Local files are referenced with the *file://* URI scheme.
 
-  - **local**: This file system is used when the scheme is *"file://"*, and it represents the file system of the local machine, including any NFS or SAN drives mounted into that local file system.
+## Pluggable File Systems
 
-  - **S3**: Flink directly provides file systems to talk to Amazon S3 with two alternative implementations, `flink-s3-fs-presto` and `flink-s3-fs-hadoop`. Both implementations are self-contained with no dependency footprint.
-    
-  - **MapR FS**: The MapR file system *"maprfs://"* is automatically available when the MapR libraries are in the classpath.  
+Apache Flink project supports the following file systems as plugins:
```

Comment:
do we have any reference _anywhere_ what a plugin is in the context of Flink?

Reply:
No, we don't have it, not yet. In this document, there is a follow up description that tells that those file systems should be copied under `plugins` dir before Flink startup to enable them.

Although, the referenced ticket focuses on documenting the plugins mechanism, I've focused mostly on updating file systems setup docs. My reasoning is the following:
 * currently only file system components are used by the plugins mechanism. Imo, it's fine for now to not have a dedicated documentation page for plugins in general. Also, there is not much to say to users about plugins mechanism, besides which Java interface it's and that they are loaded from `plugins` dir;
 * it's possible that the plugin mechanism/lifecycle may change in the future, when it's applied to other Flink components.

My proposal would be to wait when there are more use cases of plugins and when it's more refined. At the moment, I find it's scope too small to document as a separate section.